src/main/java/edu/kis/powp/jobs2d/drivers/adapter/JsonAdapter.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
```

Comment:
It seems that this class is badly located. It has nothing to do with drivers.

Reply:
Moved to command package